src/app/pages/create-account/create-account.page.ts

```diff
@@ -57,16 +62,22 @@ export class CreateAccountPage implements OnInit {
     } else {
       console.log(this.signupForm.value)
     }
-    this.showToast('Account created!');
-    err => {
-      this.showToast('There was a problem creating your account :(');
-    };
-    let navigationExtras: NavigationExtras = {
-      state: {
-        data: this.firstName
+    this.authService.signUp(this.email, this.signupForm.value.password, (error, data) => {
```

Comment:
This is the main change for this PR. We can now send a callback to the AuthService and navigate to the next page in the flow on success or stay put with an error message on failure.

Reply:
But it's just for email at the moment?